### ISSUE TITLE
https://shink.me/ 

### ISSUE THREAD

**Issue by LoboRundas** [Sainan/Universal-Bypass#1781]
After skipping it manually, it redirected me to an adshort.co that also wasn't skipped AND never actually loaded the page

**Comment by Sainan** [comment]
Example link?

**Comment by LoboRundas** [comment]
Here you go: https://shink.me/8An6F

**Comment by Sainan** [comment]
Looks like my adblocker is already skipping the timer, but I guess UB could skip the need to press the buttons. In any case, I don't think I'll be able to fix the fact that the destination link is dead/invalid.

**Comment by LoboRundas** [comment]
Makes sense, nothing you can do if the destination link isn't valid in the first place!
